### PR TITLE
[apps] Apply clang-format to modeler

### PR DESCRIPTION
**Pull request by SunBlack**

### CODE REVIEW

File: apps/modeler/src/statistical_outlier_removal_worker.cpp [hunk 1]
```diff
@@ -67,8 +67,22 @@ pcl::modeler::StatisticalOutlierRemovalWorker::initParameters(CloudMeshItem *)
 void
 pcl::modeler::StatisticalOutlierRemovalWorker::setupParameters()
 {
-  mean_k_ = new IntParameter("Mean K", "The number of nearest neighbors to use for mean distance estimation", 8, 1, 1024, 1);
-  stddev_mul_thresh_ = new DoubleParameter("Standard Deviation Multiplier", "The distance threshold will be equal to: mean + stddev_mult * stddev. Points will be classified as inlier or outlier if their average neighbor distance is below or above this threshold respectively.", 1.0, 0.1, 10, 0.1);
+  mean_k_ = new IntParameter(
```
[the remark this answers is not in the document]
Comment (SunBlack) [reply apps/modeler/src/statistical_outlier_removal_worker.cpp:70]: Any idea? Turn off clang-format for this?

File: apps/modeler/src/poisson_worker.cpp [hunk 2]
```diff
@@ -66,52 +70,68 @@ void
...
-    the octree construction is adapted to sampling density. For noise-free samples, small \
-    values in the range [1.0 - 5.0] can be used. For more noisy samples, larger values in \
-    the range [15.0 - 20.0] may be needed to provide a smoother, noise-reduced, reconstruction.",
-    poisson.getScale(), 0.1, 10.0, 0.01);
+  scale_ = new DoubleParameter(
+      "Scale",
+      "The ratio between the diameter of the cube used for reconstruction and the "
+      "diameter of the samples' bounding cube.",
+      poisson.getScale(),
+      0.1,
```
Comment (kunaltyagi) [inline apps/modeler/src/poisson_worker.cpp:113]: Maybe clang-format off for only the last 3 items

File: apps/modeler/src/parameter_dialog.cpp [hunk 3]
```diff
@@ -135,29 +134,28 @@ void
...
+pcl::modeler::Parameter*
+pcl::modeler::ParameterDelegate::getCurrentParameter(const QModelIndex& index) const
 {
   std::map<std::string, Parameter*>::iterator currentParameter = parameter_map_.begin();
 
   std::size_t currentRow = 0;
-  while(currentRow < (std::size_t) index.row() && currentParameter != parameter_map_.end()) {
-    ++ currentParameter;
-    ++ currentRow;
+  while (currentRow < (std::size_t)index.row() &&
```
Comment (kunaltyagi) [inline apps/modeler/src/parameter_dialog.cpp:155]: 🤦‍♂️ `std::find` here please (separate PR). Needs trivial issue

File: apps/modeler/src/normal_estimation_worker.cpp [hunk 4]
```diff
@@ -89,15 +89,18 @@ pcl::modeler::NormalEstimationWorker::setupParameters()
...
-  double step = range_max/1000;
+  double radius = range_max / 100;
+  double step = range_max / 1000;
 
-  search_radius_ = new DoubleParameter("Search Radius",
-    "The sphere radius that is to be used for determining the nearest neighbors", radius, 0, x_max_-x_min_, step);
+  search_radius_ = new DoubleParameter(
+      "Search Radius",
+      "The sphere radius that is to be used for determining the nearest neighbors",
+      radius,
```
Comment (kunaltyagi) [inline apps/modeler/src/normal_estimation_worker.cpp:98]: In this file too, clang-format off for parameters please

File: apps/modeler/src/main_window.cpp [hunk 5]
```diff
@@ -36,20 +36,18 @@
...
 #include <pcl/apps/modeler/main_window.h>
 
-#include <pcl/apps/modeler/scene_tree.h>
 #include <pcl/apps/modeler/dock_widget.h>
 #include <pcl/apps/modeler/render_window.h>
 #include <pcl/apps/modeler/render_window_item.h>
+#include <pcl/apps/modeler/scene_tree.h>
 
 #include <QFileInfo>
 #include <QSettings>
```
Comment (kunaltyagi) [inline apps/modeler/src/main_window.cpp:45]: Line between qt and vtk headers

File: apps/modeler/src/icp_registration_worker.cpp [hunk 6]
```diff
@@ -92,62 +92,79 @@ pcl::modeler::ICPRegistrationWorker::setupParameters()
...
-    "Maximum allowable difference between two consecutive transformations.", 0.0, 0, transformation_epsilon, step);
-
-  double euclidean_fitness_epsilon = range_max/2;
-  euclidean_fitness_epsilon_ = new DoubleParameter("Euclidean Fitness Epsilon",
-    "Maximum allowed Euclidean error between two consecutive steps in the ICP loop.", 0.0, 0, euclidean_fitness_epsilon, step);
+  double max_correspondence_distance = range_max / 2;
+  double step = range_max / 1000;
+
+  max_correspondence_distance_ =
+      new DoubleParameter("Max Correspondence Distance",
```
Comment (kunaltyagi) [inline apps/modeler/src/icp_registration_worker.cpp:99]: Same with this file and parameters

**Comment by kunaltyagi** [comment]
Should we keep this on hold till after the clang-format for headers or vice-versa?

**Comment by SunBlack** [comment]
> Should we keep this on hold till after the clang-format for headers or vice-versa?

You can continue. I just copy the `.clang-format` to this branch (without commiting). Currently my priority was just the other branch with ungrouped apps, as this is the bigger one ;-).

File: apps/modeler/src/main_window.cpp (hunk 5 above)
Comment (SunBlack) [reply apps/modeler/src/main_window.cpp:44]: Applied now automatically by clang-format  ;-)

**Comment by SunBlack** [comment]
I added this already to `format.sh`, even it is only a part of a "module", to make sure no one will break a rule until all apps are ready ;-)

**Comment by kunaltyagi** [comment]
Didn't know sub-dir would also work. Wow to bash magic...

**Comment by SunBlack** [comment]
Sure

> Didn't know sub-dir would also work. Wow to bash magic...

Of course this works, as `${PCL_DIR` already can contains `/` we just add one more via `whitelist` ;-)
https://github.com/PointCloudLibrary/pcl/blob/ada93e955bb7085646c3237a89685406f8f9a21f/.dev/format.sh#L27-L28

**Pull request merged**